### PR TITLE
RDoc-2336 [Node.js] Client API > Configuration > Read & Load balance

### PR DESCRIPTION
**Pull request by Danielle9897**
**Related issue:**
https://issues.hibernatingrhinos.com/issue/RDoc-2336/Node.js-Client-API-Configuration-Load-balance-failover
https://issues.hibernatingrhinos.com/issue/RDoc-2335/Conventions-Load-Balance-Failover-add-link-to-topology

@ml054 
Node.js - files to be reviewed:
```
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/overview.markdown
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/read-balance-behavior.js.markdown
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/load-balance-behavior.js.markdown
Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/loadBalance.js
Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/readBalance.js
```

@aviv86 
C# - files to be reviewed:
```	
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/overview.markdown
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/read-balance-behavior.dotnet.markdown
Documentation/5.2/Raven.Documentation.Pages/client-api/configuration/load-balance/load-balance-behavior.dotnet.markdown
Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/LoadBalance/ReadBalance.cs
Documentation/5.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/LoadBalance/LoadBalance.cs
```

### CODE REVIEW

**Review by aviv86 — APPROVED** [review]

File: Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/loadBalance.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        documentStore.conventions.loadBalanceBehavior = "UseSessionContext";
+
+        // Assign a method that sets the default context string
+        // This string will be used for sessions that do Not provide a context string
+        // A sample getDefaultContext method is defined below
+        documentStore.conventions.loadBalancerPerSessionContextSelector = getDefaultContext;
+        
+        // Set a seed
+        // The seed is 0 by default, provide any number to override
+        documentStore.conventions.loadBalancerContextSeed = 5
```
Comment (ml054) [inline Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/loadBalance.js:24]: ;

File: Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/readBalance.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        // ====================================================================
+
+        // Define the client configuration to put on the server
+        const configurationToSave = {
+            // Replace 'FastestNode' (from example above) with 'RoundRobin'
+            readBalanceBehavior: "RoundRobin"
+        };
+
+        // Define the put configuration operation for ALL databases
+        const putConfigurationOp = new PutServerWideClientConfigurationOperation(configurationToSave));
```
Comment (ml054) [inline Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/readBalance.js:56]: unneeed `)`?

**Review by ml054 — APPROVED** [review]

File: Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/loadBalance.js (hunk 1 above)
Comment (Danielle9897) [reply Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/loadBalance.js:24]: thanks - will fix that

File: Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/readBalance.js (hunk 2 above)
Comment (Danielle9897) [reply Documentation/5.2/Samples/nodejs/ClientApi/Configuration/LoadBalance/readBalance.js:56]: thanks - will fix that

**Review by ml054 — APPROVED** [review]

**Pull request merged**